00-base/01-colors/colors.stories.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+// import Colors from './colors';
+
+import grayscale from './colors-grayscale.js';
+import branding from './colors-branding.js';
+
+/**
+ * Storybook Definition.
+ */
+export default { title: 'Base/Colors' };
+
+// export const grayScale = () => <Colors>React Button</Colors>;
+
+const grayscaleItems = grayscale.map((item) => (
+  <li className="sg-colors__list" key={item}>
+    <div className={`sg-colors__color sg-colors__color--${item}`}></div>
+    <div className="sg-info">
+      <code>{item}</code>
+    </div>
+  </li>
+));
+const brandingItems = branding.map((item) => (
+  <li className="sg-colors__list" key={item}>
+    <div className={`sg-colors__color sg-colors__color--${item}`}></div>
+    <div className="sg-info">
+      <code>{item}</code>
+    </div>
+  </li>
+));
+
+export const Grayscale = () => <ul className="sg-colors">{grayscaleItems}</ul>;
+export const Branding = () => <ul className="sg-colors">{brandingItems}</ul>;
```

Comment:
Two small suggestions, completely non-functional. 
1. There is redundant code between grayscaleItems/brandingItems and also Grayscale/Branding. I don't see a reason for these to be defined separately. 
2. When in React-land, I like to use React Components whenever possible to get all of it's rich, juicy features. The  grayscaleItems/brandingItems functions work in this case though.
```
import React from 'react';
import grayscale from './colors-grayscale.js';
import branding from './colors-branding.js';

/**
 * Storybook Definition.
 */
export default { title: 'Base/Colors' };

const Colors = ({colors}) => (
  <ul className="sg-colors">
    { colors.map(color => (
      <li className="sg-colors__list" key={color}>
        <div className={`sg-colors__color sg-colors__color--${color}`}></div>
        <div className="sg-info">
        <code>{color}</code>
        </div>
      </li>
    ))}
  </ul>
);
export const Grayscale = () => <Colors colors={grayscale} />;
export const Branding = () => <Colors colors={branding} />;
```